app/Platform/Concerns/BuildsGameListQueries.php

```diff
@@ -34,7 +34,16 @@ private function buildBaseQuery(
         ?User $user = null,
         ?int $targetId = null,
     ): Builder {
-        $query = Game::with(['system'])
+        $query = Game::query()
```

Comment:
`query()` is just here to move `with()` to the next line so line indentation makes sense.